trytravis.py

```diff
@@ -320,9 +385,17 @@ def _slug_from_url(url):
 
 def _version_string():
     """ Gets the output for `trytravis --version`. """
+    """ Gets python version before get system information,
+    because methods like platorm.dist is not disponible
+    in python 3.8 and 3.9. """
+    py_version = sys.version[:5]
     platform_system = platform.system()
     if platform_system == 'Linux':
-        os_name, os_version, _ = platform.dist()
+        if '3.8' in py_version or '3.9' in py_version:
```

Comment:
Like I said in an earlier comment, we should be using `if hasattr(platform, "dist"):` then use `platform.dist()` otherwise fall back on `platform.system()` and `platform.version()`.

Reply:
ok, i changed